finder-app/finder-test.sh

```diff
@@ -50,14 +50,14 @@ then
 fi
 
 # natively clean and complile writer.c program
-echo "Removing the old writer utility and compiling as a native application"
-make clean
-make
+#echo "Removing the old writer utility and compiling as a native application"
+#make clean
+#make
 
 for i in $( seq 1 $NUMFILES)
 do
 	# changed from writer.sh to just writer, natively compiled writer.c
-	./writer "$WRITEDIR/${username}$i.txt" "$WRITESTR"
+	./aarch64-writer "$WRITEDIR/${username}$i.txt" "$WRITESTR"
```

Comment:
Giving a specific name to the executable shouldn't be necessary - you will want to revert back to the old way of invoking for assignment 4